app/javascript/components/world-map/WorldMap.js

```diff
@@ -0,0 +1,136 @@
+import React, { useReducer } from "react";
+import {
+  ComposableMap,
+  ZoomableGroup,
+  Geography,
+  Geographies
+} from "react-simple-maps";
+import { geoCylindricalEqualArea } from "d3-geo-projection";
+import reducer, { initialState } from "./world-map.reducer";
+import { useMarkers } from "./world-map.hooks";
+import MapBubble from "./MapBubble";
+
+const geoUrl =
+  "https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/topojson-maps/world-110m.json";
+const PetersGall = geoCylindricalEqualArea().parallel(45);
```

Comment:
The map is a bit compressed, we need to try a different value for the parallel function, maybe `30`?

Reply:
Not possible check with fausto

Reply:
Right, looks weird, but if the client wants that projection then :man_shrugging: